ThunderCloud/TSCAppDelegate.swift

```diff
@@ -183,7 +185,8 @@ open class TSCAppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificatio
 	
 	public func setupSharedUserAgent() {
 		
-		TSCRequestController.setUserAgent(TSCStormConstants.userAgent())
+        //TODO: Add support back in!
```

Comment:
Can this be done yet?

Reply:
Not yet no, but I'll get on it now as it definitely needs doing!